hub/05-view-device-data.md

```diff
@@ -0,0 +1,92 @@
+
+
+# View the Device Data
+
+Now that EdgeX Foundry is running on your Edge Node and the [Random Integer Device Service](https://docs.edgexfoundry.org/1.2/examples/Ch-ExamplesRandomDeviceService/) is posting a simple random event message every five seconds, let's explore ways to view that data.  We'll then modify the interval and range of values and see it change in reponse.
+
+## Ensure everything is functioning normally
+
+Let's open a shell to the environment that is running the Horizon Agent (Anax).  To confirm that it is the correct location and Horizon is operating properly, check the version:
+
+``` bash
+hzn version
+```
+
+It should respond with something similar to this:
+
+``` bash
+Horizon CLI version: 2.25.0
+Horizon Agent version: 2.25.0
+```
+
+Check to ensure that the `edgex_1.0.1_amd64` Service is running and your Edge Node is properly configured:
+
+``` bash
+hzn node list | jq '.configstate'
+```
+
+You should see output similar to this:
+
+``` bash
+{
+  "state": "configured",
+  "last_update_time": "2020-04-04 07:37:39 -0700 PDT"
+}
+```
+
+And last, let's ask Docker to show us the running images:
```

Comment:
You could also try the hzn service list command, that might give a more horizon like response, showing the services from the horizon perspective.  Try it and let me know what you think.

Reply:
I'd like to, but in this particular case the Horizon Service Definition groups all seven Docker images into one Horizon Service, so `hzn service list` will show only:
``` json
[
  {
    "url": "com.github.joewxboy.horizon.edgex",
    "org": "mycluster",
    "version": "1.0.1",
    "arch": "amd64",
    "variables": {}
  }
]
```
This is counter-productive since I am attempting to demonstrate that all of the Docker images are actually running and stable, and are not restarting.

Reply:
But yes, I can add a section showing that Horizon knows that the Service is running _before_ I drill down into individual Docker images.